presidio-anonymizer/presidio_anonymizer/entities/engine/anonymizer_result.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Dict
+
+from presidio_anonymizer.entities.engine import PIIEntity
+from presidio_anonymizer.entities.engine.result import OperatorResult
+
+
+class AnonymizerResult(PIIEntity):
+    """Information about the encrypted entity."""
+
+    def __init__(self, start: int,
+                 end: int, entity_type: str) -> 'AnonymizerResult':
+        """
+        Have the information about the entity we already encrypted and want to decrypt.
```

Comment:
is this class only for decrypt purpose? this comment is not clear

Reply:
Yes, changing the documentation